addon/options.html

```diff
@@ -6,107 +6,26 @@
   <head>
     <meta charset="utf-8">
     <link rel="stylesheet" href="options.css" type="text/css">
+    <!-- React and friends must be loaded before the ui.js module -->
+    <script type="text/javascript"
+      src="./content/vendor/react.js"></script>
```

Comment:
Are we not able to import these as modules? I'm wondering if we could actually get to the stage where options.js is the only script statement here, and then everything is imported into that via import statements?

This would certainly be nicer from the viewpoint of handling globals.

Reply:
Done